Doc/library/imp.rst

```diff
@@ -51,7 +51,9 @@ This module provides an interface to the mechanisms used to implement the
    places are searched: the function tries to find a built-in module with the
    given name (:const:`C_BUILTIN`), then a frozen module (:const:`PY_FROZEN`),
    and on some systems some other places are looked in as well (on Windows, it
-   looks in the registry which may point to a specific file).
+   looks in the registry which may point to a specific file). If the path in
```

Comment:
This paragraph does not need any changes.  This information is already covered below in the paragraph about the return values.

Reply:
Do you mean 
> If the module does not live in a file, the returned file is None, pathname is the empty string,
But is should return None for both file and file_path (not empty string)

Reply:
Probably yes, I agree with Eric, let's keep all "return value" information in the same paragraph and keep this paragraph focused only on general behavior.